### PR TITLE
clean up new user registration page a bit

### PR DESCRIPTION
**Pull request by s-cheng**
This PR is just cleaning up the styling a bit on the new user registration page (the one where you pick your username). This should basically not affect LW.

Old, desktop:
Screen Shot 2023-08-08 at 10 31 57 AM

New, desktop:
Screen Shot 2023-08-08 at 10 52 07 AM

-----
Old, mobile:
Screen Shot 2023-08-08 at 10 54 46 AM

New, mobile:
Screen Shot 2023-08-08 at 10 59 48 AM

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1205234270505745) by [Unito](https://www.unito.io)

### CODE REVIEW

**Comment by s-cheng** [comment]
@agnesstenlund FYI I just cleaned up this page a bit, since I was registering a new user and didn't like how the text looked here - feel free to make more adjustments :)

File: packages/lesswrong/components/users/NewUserCompleteProfile.tsx [hunk 1]
```diff
@@ -28,6 +35,10 @@ const styles = (theme: ThemeType): JssStyles => ({
       color: theme.palette.grey[600],
     },
   },
+  sectionHeadingText: isEAForum ? {
```
Comment (jpaddison3) [inline packages/lesswrong/components/users/NewUserCompleteProfile.tsx:38]: I don't think less wrong uses this page at all? So shouldn't be necessary — just make it good for anyone using it, IMO.

**Review by jpaddison3 — APPROVED** [review]
ASTLAC

File: packages/lesswrong/components/users/NewUserCompleteProfile.tsx (hunk 1 above)
Comment (s-cheng) [reply packages/lesswrong/components/users/NewUserCompleteProfile.tsx:38]: Oh interesting, I didn't know that, let me check with them

**Comment by s-cheng** [comment]
Sounds like LW doesn't use this, so I removed a bunch of forum-gating. Also I noticed that this page can have different backgrounds depending on which URL you hit (maybe that shouldn't happen but don't want to fix it now), so I updated it to account for the grey background version:

Screen Shot 2023-08-08 at 4 58 36 PM

File: packages/lesswrong/components/users/NewUserCompleteProfile.tsx [hunk 2]
```diff
@@ -146,7 +161,9 @@ const NewUserCompleteProfile: React.FC<NewUserCompleteProfileProps> = ({ current
       
       {/* Facebook user with no email fix (very small % of users) */}
```
Comment (s-cheng) [inline packages/lesswrong/components/users/NewUserCompleteProfile.tsx:162]: @jpaddison3 Does this Facebook case apply to us? If not, then could we remove this code?

**Pull request merged**